inference-engine/ie_bridges/java/cpp/blob.cpp

```diff
@@ -0,0 +1,143 @@
+#include <inference_engine.hpp>
+#include "openvino_java.hpp"
+#include "jni_common.hpp"
+
+using namespace InferenceEngine;
+
+JNIEXPORT jfloatArray JNICALL Java_org_intel_openvino_Blob_asFloat(JNIEnv *env, jobject obj, jlong addr)
+{
+    static const char method_name[] = "asFloat";
+
+    try
+    {
+        Blob::Ptr *output = reinterpret_cast<Blob::Ptr *>(addr);
+        const float *buffer = as<MemoryBlob>(*output)->rmap().as<const float *>();
```

Comment:
It is not correct usage. you should hold LockedMemory. It is required to work with RemoteBlob.